### PR TITLE
Project Week 2 

### PR DESCRIPTION
**Pull request by jessika-hage**

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,437 @@
...
+  color: white;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.links:hover {
+  color: #DABED5;
+}
+
+/* Navigation bar and dropdown for mobile phones */
```
Comment (JennieDalgren) [inline code/style.css:110]: good job with comments like this!

**Review by JennieDalgren — CHANGES_REQUESTED** [review]
Really nice job with this project!

The only thing I would like you to change to be approved is the indentation in the html file. Also try to use more semantic html tags, not only divs 😉
Yes, we're a bit picky with tidying the code. That's just so that you'll get into good habit of writing neat and clean code!

Take a look at [our code guidelines.](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e)
We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

All in all a very good project!
Keep up the good work 🥇

**Comment by jessika-hage** [comment]
I think I fixed the needed changes now! :)

File: code/style.css [hunk 2]
```diff
@@ -27,10 +27,7 @@ body {
 .site-name .logo {
   width: 35px;
   margin-right: 10px;
-  animation-name: logo;
-  animation-duration: 1s;
-  animation-iteration-count: infinite;
-  animation-direction: alternate-reverse;
+  animation: logo 1s alternate-reverse infinite;
```
Comment (JennieDalgren) [inline code/style.css:30]: ⭐

**Review by JennieDalgren — APPROVED** [review]
💯

**Pull request closed without merging**